src/services/permanence-conseillers/permanence/repositories/permanence-conseiller.repository.js

```diff
@@ -168,6 +168,15 @@ const deleteCraPermanence = db => async permanenceId => {
   });
 };
 
+const deleteCraConseillerPermanence = db => async (permanenceId, idConseiller) => {
+  await db.collection('cras').updateMany({
+    'permanence.$id': new ObjectId(permanenceId),
+    'conseillers': { $in: [new ObjectId(idConseiller)] }
```

Comment:
$in idsConseillerList ? ou $eq idConseiller ?

Reply:
plutôt $in car `conseillers` est un array

Reply:
c'est conseiller qu'il y a dans la collection cras non ?

Reply:
Ah vraiment bien vue !! j'ai confondue, oui du coup eq pour conseiller.$id